consensus/istanbul/backend/backend.go

```diff
@@ -301,13 +301,15 @@ func (sb *Backend) SendDelegateSignMsgToProxy(msg []byte) error {
 
 // SendDelegateSignMsgToProxiedValidator sends an istanbulDelegateSign message to a
 // proxied validator if one exists
-func (sb *Backend) SendDelegateSignMsgToProxiedValidator(msg []byte) error {
+func (sb *Backend) SendDelegateSignMsgToProxiedValidator(msg []byte) {
```

Comment:
Is there a specific reason that you removed the returned error?

Reply:
yes, this method it called async, the error was never handled during async handling. So i decided to rather log the error instead of just hiding it by not handling it at all. Its not good, but its better than it was before.